downloads/jdkmon/readme.md

```diff
@@ -24,24 +22,24 @@ a menu will appear where you can select
 
 `JDK Main: The main window`
 
-`Rescan: Will rescan for installed JDK's and check for updates`
+`Rescan: Will rescan for installed JDKs and check for updates`
 
 `Search path: Will open the directory chooser to select the search path`
 
 `Exit: Exit the application`
 `
 
-The main window will show you all JDK's found by JDKMon and if there is an
+The main window will show you all JDKs found by JDKMon and if there is an
 update available it will show you the archive types of the available updates.
 In the image below you can see that there is an update for GraalVM available
 and that you can download it as a tar.gz file.
 To download an update just click on the archive type tag and choose a folder
 where the download should be stored.
 
 How it looks on MacOS and Linux:
-![Screenshot](https://github.com/HanSolo/JDKMon/raw/main/screenshot.png)
+![Screenshot](https://github.com/HanSolo/jfxcentral-data/raw/JDKMon/downloads/jdkmon/banner.jpg)
```

Comment:
Almost .... :-) the banner.jpg file is in the same folder, right? So you do not need the whole path, "banner.jpg" is enough.